x-pack/plugins/security_solution/common/endpoint/utils/commands.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  ConsoleResponseActionCommands,
+  ResponseActionsApiCommandNames,
+} from '../service/response_actions/constants';
+import { RESPONSE_CONSOLE_ACTION_COMMANDS_TO_REQUIRED_AUTHZ } from '../service/response_actions/constants';
+import type { EndpointPrivileges } from '../types';
+
+export const getRbacControl = ({
+  commandName,
+  privileges,
+}: {
+  commandName: ConsoleResponseActionCommands;
+  privileges: EndpointPrivileges;
+}): boolean => {
+  return Boolean(privileges[RESPONSE_CONSOLE_ACTION_COMMANDS_TO_REQUIRED_AUTHZ[commandName]]);
+};
+
+/**
+ * map actual command to ui command
+ * unisolate -> release
+ * running-processes -> processes
+ */
+export const getUiCommand = (
```

Comment:
We have a `const` that does this mapping here: https://github.com/elastic/kibana/blob/f9f4c1a8009e03f6e0f46879dc912b7baa9e3a9f/x-pack/plugins/security_solution/common/endpoint/service/response_actions/constants.ts#L87-L98

That being said, and following from my earlier comment, I don't think you need this for the server side.